newjson.py

```diff
@@ -0,0 +1,89 @@
+import json
+import time
+
+phonebook = {}
+
+with open("json1.json") as rfile:
```

Comment:
nit: better to write which mode is used for opening files

```suggestion
with open("json1.json", "r") as rfile:
```